cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -148,4 +184,5 @@ def describeType(DescribableHelper.ParameterType type, int headerLevel) throws E
             assert false: type;
         }
     }.call()
-}
\ No newline at end of file
+}
+st.adjunct(includes: 'org.jenkinsci.plugins.workflow.cps.Snippetizer.js.workflow')
```

Comment:
Looks wrong; what includes the CSS? I think you meant to use an adjunct name like `org.jenkinsci.plugins.workflow.cps.Snippetizer.reference` with resources named `org/jenkinsci/plugins/workflow/cps/Snippetizer/reference.js` and `org/jenkinsci/plugins/workflow/cps/Snippetizer/reference.css`.